numpy/core/tests/test_umath.py

```diff
@@ -1422,6 +1422,18 @@ def __array_wrap__(self, arr, context):
         a = A()
         self.assertRaises(RuntimeError, ncu.maximum, a, a)
 
+    def test_none_wrap(self):
+
+        class A(object):
+            def __array__(self):
+                return np.zeros(1)
+
+            def __array_wrap__(self, arr, context):
```

Comment:
This is not the only place where this is missing in this file, so I'll leave the others to keep this patch small